.travis.yml

```diff
@@ -7,19 +7,29 @@ python:
 env:
   global:
     - ASYNC_TEST_TIMEOUT=15
+    - ETCDCTL_API=3
 
 # installing dependencies
 before_install:
   - pip install --upgrade setuptools pip
   - pip install --pre -r dev-requirements.txt
 install:
   - pip install --pre -r dev-requirements.txt .
-  # TODO: setup etcd
+  - pip install pytest-travis-fold
+  - pip install requests
+  - pip install etcd3
+  - pip install asyncio
```

Comment:
Let's add these to `dev-requirements.txt` if they are only for testing and `requirements.txt` if they are required to run the program itself.

Additionally, asyncio is in the standard library, it doesn't need to be pip installed.